<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XUVideoPlayer</class>
 <widget class="QWidget" name="XUVideoPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="VideoplayerreturnButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="VideoplayerbaseWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>520</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="VideoplayerplaySlider">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>520</y>
     <width>1022</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="VideoplayerplayButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>540</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="VideoplayernextButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>540</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="VideoplayerVolumedownButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>540</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="VideoplayervolumeSlider">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>560</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="VideoplayerVolumeupButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>540</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="VideoplayerFullscreenButton">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>540</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="VideoplayerplayLabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>560</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="VideoplayerdurationLabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>560</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>/00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <zorder>VideoplayerbaseWidget</zorder>
  <zorder>VideoplayerreturnButton</zorder>
  <zorder>VideoplayerplaySlider</zorder>
  <zorder>VideoplayerplayButton</zorder>
  <zorder>VideoplayernextButton</zorder>
  <zorder>VideoplayerVolumedownButton</zorder>
  <zorder>VideoplayervolumeSlider</zorder>
  <zorder>VideoplayerVolumeupButton</zorder>
  <zorder>VideoplayerFullscreenButton</zorder>
  <zorder>VideoplayerplayLabel</zorder>
  <zorder>VideoplayerdurationLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
